rules/windows/execution_from_unusual_directory.toml

```diff
@@ -45,3 +45,14 @@ process.executable : ("C:\\PerfLogs\\*.exe","C:\\Users\\Public\\*.exe","C:\\User
  /* and not (process.code_signature.subject_name == "Microsoft Corporation" and process.code_signature.trusted == true) */
 '''
 
+[[rule.threat]]
```

Comment:
maybe https://attack.mitre.org/techniques/T1036/005/ is more appropriate ?

Reply:
agreed, adjusted.